docs/writing_wiki.md

```diff
@@ -27,12 +27,12 @@ The wiki can be run locally by doing the following in a terminal:
     * if not, check ``pip3 -V``
 3. Set up a virtual environment (steps assume that your current dir is project root) 
       1. Install virtualenv with ``pip install virtualenv``
-            * else ``pip3 install virtualenv``
+        * else ``pip3 install virtualenv``
       2. run ``python -m venv venv``
-            * else ``python3 -m venv venv``
-      3. source the virtual environment
-            * Linux:    ``source venv/bin/activate`` 
-            * Windows:  ``.\venv\Scripts\activate.bat``
+        * else ``python3 -m venv venv``
+      1. source the virtual environment
```

Comment:
It says 1. source, but should be  3. source. Maybe use a, b and c instead of numbers?